Apps/Src/CAN_Queue.c

```diff
@@ -20,25 +20,21 @@ static OS_MUTEX canFifo_Mutex;
 
 void CAN_Queue_Init(void) {
     OS_ERR err;
-    CPU_TS ticks;
     RTOS_BPS_MutexCreate(&canFifo_Mutex, "CAN queue mutex");
     OSSemCreate(&canFifo_Sem4,
                 "CAN queue semaphore",
                 0,
                 &err);
     assertOSError(err);
-    OSMutexPend(&canFifo_Mutex, 0, OS_OPT_POST_NONE, &ticks, &err);
-    assertOSError(err);
+    RTOS_BPS_MutexPend(&canFifo_Mutex, 0, OS_OPT_POST_NONE);
```

Comment:
Lmao I just realized we've been calling it like this the whole time and `OS_OPT_POST_NONE` isn't evan a valid option for `OSMutexPend`. See the micrium docs here for the proper options: [https://docs.silabs.com/micrium/latest/micrium-kernel-api/05-kernel-mutex-api#os-mutex-pend](https://docs.silabs.com/micrium/latest/micrium-kernel-api/05-kernel-mutex-api#os-mutex-pend). I think we probably want to use `OS_OPT_PEND_BLOCKING` for everything